packages/components/inputs/checkbox-input/src/checkbox.tsx

```diff
@@ -36,26 +56,10 @@ const Checkbox = (props) => {
       readOnly={props.isReadOnly}
       checked={props.isChecked && !props.isIndeterminate}
       onChange={handleChange}
-      ref={ref}
-      /* ARIA */
-      aria-readonly={props.isReadOnly}
-      aria-checked={props.isChecked}
-      {...props}
```

Comment:
Is removing this prop spreading on purpose?

Reply:
to me, it does.
We pass in the props we accept on the component and the rest are `aria/data` attributes we forward.
then there isn't any leak from what I can tell.

Reply:
I am thinking if someone is leveraging this props spreading in their code base, it might result in breaking change.

Reply:
and they shouldnt.